src/pages/index.js

```diff
@@ -77,7 +77,7 @@ export default function Dashboard({ data: { articleImages, quoteImage } }) {
         ))}
       </Row>
       <QuoteWithImage quote={quote} />
-      <EquitySection />
+      <EquitySection location={location} />
```

Comment:
see note re: this prop. at any rate the section component should default to Main, so shouldn't need any props